src/redux/actions/logOut.actions.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-undef */
+/* eslint-disable import/prefer-default-export */
+export function logOutUser() {
+  sessionStorage.removeItem('jwt');
```

Comment:
Hey @EmyRukundo, I don't think we have `jwt` in our sessionStorage. I'm wondering why are you using it here.

Reply:
Thanks @eliemugenzi , will correct it